src/caret_analyze/architecture/struct/callback.py

```diff
@@ -101,13 +105,27 @@ def subscribe_topic_name(self) -> Optional[str]:
         return self._subscribe_topic_name
 
     @property
-    def publish_topic_names(self) -> Optional[Tuple[str, ...]]:
+    def publish_topic_names(self) -> Optional[List[str]]:
         return self._publish_topic_names
 
     @abstractmethod
     def to_value(self) -> CallbackStructValue:
         pass
 
+    def rename_node(self, src: str, dst: str):
```

Comment:
```suggestion
    def rename_node(self, src: str, dst: str) -> None:
```

Reply:
Fixed it in [b1f7b1b](https://github.com/tier4/CARET_analyze/pull/156/commits/b1f7b1b50f5be951a07c19b8ae13496b78a7ebee)